docs/topics/Gateway.md

```diff
@@ -94,6 +94,7 @@ def on_websocket_message(msg):
 |-------|------|-------------|
 | v | integer | Gateway Version to use |
 | encoding | string | 'json' or 'etf' |
+| compress | string | 'zlib-stream' (optional) |
```

Comment:
Optional is denoted with the tailing `?` after the field name everywhere else

Reply:
I believe all query strings are optional.

Reply:
Hmm, couldn't we leave it out and note it below the table ? Or should we add those `?` for all three fields ?